packages/synapse-interface/pages/bridge/BridgeWatcher/index.tsx

```diff
@@ -12,6 +12,8 @@ import BridgeEvent from './BridgeEvent'
 import { BridgeWatcherTx } from '@types'
 import { GETLOGS_SIZE, GETLOGS_REQUEST_COUNT } from '@constants/bridgeWatcher'
 import { useSynapseContext } from '@/utils/providers/SynapseProvider'
+import { useSelector } from 'react-redux';
+import { RootState } from '../../../store/store'
```

Comment:
we've got this alias'd:

```
import { RootState } from '@/store/store'
```

Reply:
https://github.com/synapsecns/sanguine/pull/1046/commits/2497da4d9330ab7886d18fb6f93e1f211caee9e2